generator/app.go

```diff
@@ -55,13 +60,93 @@ const (
 
 	regexRawGitHubURL = "https://raw.githubusercontent.com/(?P<org>[^/]+)/(?P<repo>[^/]+)/(?P<branch>[^/]+)/(?P<path>.*)"
 	regexGitHubURL    = "https://github.com/(?P<org>[^/]+)/(?P<repo>[^/]+)/(blob|tree)/(?P<branch>[^/]+)/(?P<path>.*)"
+
+	// For KEPs automation
+	alpha  = "alpha"
+	beta   = "beta"
+	stable = "stable"
```

Comment:
These values are defined in https://github.com/kubernetes/enhancements/blob/9d63597d2d85a0ffba5bb4cee0172157948168ce/api/proposal.go#L35-L39 as well. Can we please use them as they are the source of truth for the manifest?

Reply:
Thanks for pointing. Resolved in the refreshed commit. :)